WRT-230 New group after release (#29)
tradingview/rest_integrations_docs: source/data/Groups.rst

      ]
    }
  }

Adding groups after release
............................

> To add a new symbol group after release, implement the groups and add them into the staging environment.

Reviewer: Let's use either singular or plural forms for consistency.
```suggestion
To add a new symbol group after release, implement the group and add it into the staging environment.
```

> To add a new symbol group after release, implement the groups and add them into the staging environment.

Reviewer: ```suggestion
To add new symbol groups after release, implement the groups and add them into the staging environment.
```
